tests/handlers/test_register.py

```diff
@@ -48,6 +49,13 @@ def setUp(self):
         ])
 
         hs.get_handlers().auth_handler = self.mock_handler
+        self.store = hs.get_datastore()
+        self.frank = UserID.from_string("@frank:test")
+
+        yield self.store.register(
+            user_id=self.frank.to_string(),
+            token="jkv;g498752-43gj['eamb!-5",
+            password_hash=None)
```

Comment:
Would it be better for this to be in `test_if_user_exists` rather than in setup?